src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/__tests__/ExposureDetectionViewModelTests.swift

```diff
@@ -898,20 +898,20 @@ class ExposureDetectionViewModelTests: XCTestCase {
 
 		// Explanation section
 		section = dynamicTableViewModel.section(5)
-		XCTAssertEqual(section.cells.count, hasAtLeastOneDayWithLowRiskLevel ? 3 : 2)
+		XCTAssertEqual(section.cells.count, 2)
 		XCTAssertEqual(section.cells[0].cellReuseIdentifier.rawValue, "headerCell")
 		XCTAssertEqual(section.cells[1].cellReuseIdentifier.rawValue, "labelCell")
 
-		if hasAtLeastOneDayWithLowRiskLevel {
-			XCTAssertEqual(section.cells[2].cellReuseIdentifier.rawValue, "linkCell")
-
-			switch section.cells[2].action {
-			case .open(url: let url):
-				XCTAssertEqual(AppStrings.ExposureDetection.explanationFAQLink, url?.absoluteString)
-			default:
-				XCTFail("FAQ Link cell not found")
-			}
-		}
+//		if hasAtLeastOneDayWithLowRiskLevel {
+//			XCTAssertEqual(section.cells[2].cellReuseIdentifier.rawValue, "linkCell")
+//
+//			switch section.cells[2].action {
+//			case .open(url: let url):
+//				XCTAssertEqual(AppStrings.ExposureDetection.explanationFAQLink, url?.absoluteString)
+//			default:
+//				XCTFail("FAQ Link cell not found")
+//			}
+//		}
```

Comment:
Do we still need this?

Reply:
of course not. Thought I had it removed already...

Reply:
of course not. Thought I has it removed already...